spec/ysh-blocks.test.sh

```diff
@@ -92,6 +92,9 @@ proc my-cd (; b) {
   # Yup, this is a problem
   var s = io.captureStdout(b)
   echo "stdout $s"
+
+  setvar s = io.captureOutputs(b)
+  echo "stdout $[s.stdout]"
```

Comment:
this is a bit confusing because it doesn't change any of the assertions ?  what happened?

Reply:
Soo turns out this test was marked as "allowed failure" because it didn't work in the past. But it does work now, so I added the proper `STDOUT`. I think it's behaviour is correct now.